plugins/reports/counter/classes/CounterReport.inc.php

```diff
@@ -0,0 +1,301 @@
+<?php
+
+/**
+ * @file plugins/reports/counter/classes/CounterReport.inc.php
+ *
+ * Copyright (c) 2014 University of Pittsburgh
+ * Distributed under the GNU GPL v2 or later. For full terms see the file docs/COPYING.
+ *
+ * @class CounterReport
+ * @ingroup plugins_reports_counter
+ *
+ * @brief A COUNTER report, base class
+ */
+require_once(dirname(dirname(__FILE__)).'/classes/COUNTER/COUNTER.php');
+
+define('COUNTER_EXCEPTION_WARNING', 0);
+define('COUNTER_EXCEPTION_ERROR', 1);
+define('COUNTER_EXCEPTION_PARTIAL_DATA', 4);
+define('COUNTER_EXCEPTION_NO_DATA', 8);
+define('COUNTER_EXCEPTION_BAD_COLUMNS', 16);
+define('COUNTER_EXCEPTION_BAD_FILTERS', 32);
+define('COUNTER_EXCEPTION_BAD_ORDERBY', 64);
+define('COUNTER_EXCEPTION_BAD_RANGE', 128);
+define('COUNTER_EXCEPTION_INTERNAL', 256);
+
+define('COUNTER_CLASS_PREFIX', 'CounterReport');
+
+// COUNTER as of yet is not internationalized and requires English constants
+define('COUNTER_LITERAL_ARTICLE', 'Article');
+define('COUNTER_LITERAL_JOURNAL', 'Journal');
+define('COUNTER_LITERAL_PROPRIETARY', 'Proprietary');
+
+class CounterReport {
+
+	var $parentPluginCategory;
+	var $parentPluginName;
+	
+	var $_release;
```

Comment:
sorry, not sure if it wasn't here before, but the documentation is missing for those members...